notebooks/documentation/prerequisites_azure_resources.ipynb

```diff
@@ -0,0 +1,67 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Prerequisites - Azure OpenAI and Cognitive Services resources\n",
+    "\n",
+    "In order to use SynapseML OpenAI feature or the Cognitive Services feature, specific Azure resources are required. This documentation walk you through the process of setting up these resources and acquiring the necessary credentials.\n",
+    "\n",
+    "To utilize the SynapseML OpenAI feature or the Cognitive Services feature, you need to create resources and obtain the API key."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Azure OpenAI resource\n",
+    "\n",
+    "The Azure OpenAI service can be used to solve a large number of natural language tasks through prompting the completion API. To make it easier to scale your prompting workflows from a few examples to large datasets of examples, we have integrated the Azure OpenAI service with the distributed machine learning library SynapseML. This integration makes it easy to use the Apache Spark distributed computing framework to process millions of prompts with the OpenAI service."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "openai-prerequisites"
+    ]
+   },
+   "source": [
+    "To Create a new Azure OpenAI Resource and obtain an OpenAI API key, you need\n",
```

Comment:
the prereqs should be the same moreorless for azure openai and cog services, things like azure accounts will be needed for both